kqueen/auth.py

```diff
@@ -16,10 +15,11 @@ def authenticate(username, password):
         user: authenticated user
 
     """
+    from kqueen.server import bcrypt
```

Comment:
Is there a reason to import it here and not at the the beginning?

Reply:
I think there was some circular import problem, import failed at the beginning of file.